packages/prop-controllers/src/navigation-links/navigation-links.ts

```diff
@@ -230,19 +233,23 @@ export function getNavigationLinksPropControllerSwatchIds(
   if (data == null) return []
 
   const value = getNavigationLinksPropControllerDataNavigationLinksData(data)
+  const swatchIds = (color: ResponsiveColorData | undefined) =>
+    color?.map((override) => override.value).map((color) => color.swatchId) ??
+    []
 
   return (
-    value?.flatMap((item) => {
-      switch (item.type) {
+    value?.flatMap(({ type, payload }) => {
+      switch (type) {
         case 'button':
+          return [...swatchIds(payload.color), ...swatchIds(payload.textColor)]
+
         case 'dropdown':
           return [
-            ...(item.payload.color
-              ?.map((override) => override.value)
-              .map((color) => color.swatchId) ?? []),
-            ...(item.payload.textColor
-              ?.map((override) => override.value)
-              .map((color) => color.swatchId) ?? []),
+            ...swatchIds(payload.color),
+            ...swatchIds(payload.textColor),
+            ...(payload.links?.flatMap((link) =>
+              swatchIds(link.payload.color),
```

Comment:
Introspect dropdown links.

Reply:
This bug goes a long way back: https://github.com/makeswift/makeswift/blame/9ee292e4c578d7dae7dfbe58a3aaca84b2f914a7/packages/runtime/src/prop-controllers/introspection.ts#L182